src/main/resources/i18n/messages.properties

```diff
@@ -94,6 +94,7 @@ email.notification.aux.difficulty.medium=Medium
 email.notification.aux.difficulty.hard=Hard
 
 # Plagiarism
+email.plagiarism.title=Exercise {0} in the course {1}: New Plagiarism Case
```

Comment:
I'd prefer`New Plagiarism Case: Exercise "{0}" in the course "{1}"` but that is a matter of taste

Reply:
Changed the title according to your suggestion. It looks better. Thanks!